en-ar/jw300-tedtalk-baseline/notebook.ipynb

```diff
@@ -0,0 +1,1370 @@
+{
```

Comment:
<p>This seems to be a personal file path and won't generalize to other people's systems - this should be captured in the $gdrive_path variable</p><p> <code>/GW/Health-Corpus/work/masa/$gdrive_path</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
rsolved